app/components/moj_forms/back_link_component.rb

```diff
@@ -0,0 +1,29 @@
+class MojForms::BackLinkComponent < GovukComponent::Base
```

Comment:
no general consensus as namespacing  like this, but the rest of the app does namespacing like so:

```
module MojForms
  class BackLinkComponent < GovukComponent::Base
  end
end
```

Reply:
I defaulted to writing it like that, but in the midst of troubleshooting, I opted to directly copy GovukComponents just to make sure... I think I actually prefer having the `module` defnition in there.